packages/components/src/dashicon/index.js

```diff
@@ -913,6 +914,7 @@ export default class Dashicon extends Component {
 				width={ size }
 				height={ size }
 				viewBox="0 0 20 20"
+				{ ...colorProp }
```

Comment:
It'd be simpler to express this as just `color={ color }`, skipping all of the `colorProp` conditions and object spread.

Further, it makes me wonder why not just pass through all additional props to the SVG.

Further further, these edits are made a generated file, where the source lives at https://github.com/WordPress/dashicons . See the note at the top of the file.

Reply:
Thanks @aduth for the comments!

> It'd be simpler to express this as just `color={ color }`

I did try that, but when `color` is undefined, it would remove the default color we are using for the rest of the icons. (for some reason)

With @Tug we decided to pass a `style` prop, and extract color from there, with the added advantage of passing more style props, that we are now taking advantage of. Please check the last PR :) 

> these edits are made a generated file

I was wondering about that. So I should edit https://github.com/WordPress/dashicons/tree/master/sources/react too 🤔